README.md

```diff
@@ -40,15 +39,53 @@ important concepts to help understanding the internals of the SDK:
   accessed through. Most operations are of the form `emsdk command`. To access
   the emsdk script, launch the Emscripten Command Prompt.
 
+## System Requirements
```

Comment:
this section duplicates docs on our main site. how about linking to there, so there's a single source of truth?

Reply:
I see.  For building from source source the dependencies are listed in https://emscripten.org/docs/building_from_source/toolchain_what_is_needed.html.  For the the dependencies of the emsdk itself I think this is the only place we document those, no?

Reply:
Yes, we could document the emsdk specific dependencies here, good point. But the only dependency there is python, I think? Can remove java below.

Reply:
But more generally, I don't think we need a specific section per OS? Just "the emsdk itself requires python, as described in the emscripten docs. The other requirements of emscripten itself are also in those docs."

Reply:
Sure I will do that.  But we do also depend on java, since emsdk doesn't handle install that.   Ideally emsdk users should not need to look at `building_from_source/toolchain_what_is_needed.html` since they are not building from source right?

Reply:
Another way of putting it:  We are document here any emscripten dependencies that emsdk doesn't install automatically.  At least that is how i think of it.

Reply:
I think we do install java, don't we? It's in the emsdk_manifest file.

But to the second point, yeah, I think it makes sense to document here any deps emcc needs that the emsdk doesn't provide. I don't think there should be any, though, aside from python (which the emsdk doesn't provide as it needs it itself)?

Reply:
Oh, looks like we install java on windows but nowhere else. So yeah, maybe mentioning that java is needed for closure is ok.